src/screens/AdminPortal/OrgList/OrgList.tsx

```
@@ -1,74 +1,39 @@
+import type { ChangeEvent } from 'react';
+import NotificationIcon from 'components/NotificationIcon/NotificationIcon';
+import OrganizationCard from 'shared-components/OrganizationCard/OrganizationCard';
+import EmptyState from 'shared-components/EmptyState/EmptyState';
+import style from './OrgList.module.css';
+import { Group, Search } from '@mui/icons-material';
+import SearchFilterBar from 'shared-components/SearchFilterBar/SearchFilterBar';
+import { Alert } from 'react-bootstrap';
+import BaseModal from 'shared-components/BaseModal/BaseModal';
 
-const { getItem } = useLocalStorage();
+const { getItem, setItem, removeItem } = useLocalStorage();
```

Biome (2.1.2)

[error] This hook is being called conditionally, but all hooks must be called in the exact same order in every component render.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)